lib/serializables/log-event.js

```diff
@@ -51,9 +51,6 @@ function LogEventFactory (
             host: {
                 type: 'string'
             },
-            module: {
```

Comment:
@yyscamper xiaofeifei, At first I had the same thought as you did. Then when I looked deeper into the logs, I found the log anyway will give the file name and line number. So removing module names won't affect the analysis, but makes the log less.
1|http     | [ debug  ] [2017-08-09T19:23:30.688Z] advertise-alive
1|http     |  -> /lib/services/upnp-service.js:103